build_tools/kokoro/gcp_ubuntu/cmake-bazel/linux/x86-swiftshader/build.sh

```diff
@@ -45,6 +46,7 @@ BAZEL_BINDIR="$(${BAZEL_CMD[@]?} info bazel-bin)"
       --config=generic_clang \
       --test_tag_filters="-nokokoro" \
       --build_tag_filters="-nokokoro"
+bash ./symlink_binaries.sh
```

Comment:
So this is nice for exactly the use case of building the binaries yourself, but it seems like it won't work well for "bring your own binary", which was why we had it the other way before. I like people being able to specify where they put the binaries. It could just be the PATH though

Reply:
This is just to make the Python packages line up, and since our integration tests are using the development Python packages here, we have to set them up to work. I expect that most folks who are bringing their own binary will either install the relevant Python package (which will provide it) or add it to the path. May need a few more tweaks to make that seamless. See, for example, that the benchmark CMake is just doing `find_program(..., iree-import-tflite)`.